pages/index.html

```diff
@@ -1,12 +1,29 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/styles/index_styles.css">
 </head>
+
 <body>
-  
+    <header>
+        <h1> Ashley Benson</h1>
+        <nav>
+            <a href="http://127.0.0.1:5500/pages/about.html" id="about">About</a>
```

Comment:
Since all the pages are in the same directory, we could link among them using relative paths

```html
            <a href="about.html" id="about">About</a>
```

or if we used absolute paths, we should leave off the server info

```html
            <a href="/pages/about.html" id="about">About</a>
```